### ISSUE TITLE
Contact Form Submission - Suggestion (Persistent Data Structures)

### ISSUE THREAD

**Issue by maggieliu05** [cpinitiative/usaco-guide#3118]
Someone submitted the contact form!

**URL**: https://usaco.guide/adv/persistent?lang=cpp
**Module**: Advanced - Persistent Data Structures
**Topic**: Suggestion
**Message**: 
Closest Equals problem from cf is solvable with normal segment tree

**Comment by bqi343** [comment]
True, this problem (and application 2) probably shouldn't be part of the module. Also, for application 1 it should be noted that
 - persistent segment tree is needed only if the queries are online.
 - persistent segment tree isn't necessarily harder to implement than 2D Fenwick tree, and it's also faster ( $O(\log N)$ vs $O(\log^2N)$ queries).
